packages/realm/src/platform/node/fs.ts

```diff
@@ -78,7 +78,7 @@ inject({
         dirent.name.endsWith(".realm.fresh.lock") ||
         dirent.name.endsWith(".realm.log")
       ) {
-        unlinkSync(direntPath);
```

Comment:
I included this, because the tests kept failing locally from:

```
  ● useQuery › returns the same collection reference if there are no changes

    ENOENT: no such file or directory, lstat '/Users/kraen.hansen/Repositories/realm-js/packages/realm-react/default.realm.note'

      79 |         dirent.name.endsWith(".realm.log")
      80 |       ) {
    > 81 |         rmSync(direntPath);
         |               ^
      82 |       }
      83 |     }
      84 |   },

      at Object.removeRealmFilesFromDirectory (../realm/src/platform/node/fs.ts:81:15)
      at Function.removeRealmFilesFromDirectory [as clearTestState] (../realm/src/Realm.ts:194:8)
      at Object.clearTestState [as cleanup] (src/__tests__/createRealmTestContext.ts:64:13)
      at Object.cleanup (src/__tests__/useQueryHook.test.tsx:70:13)
```